### PR TITLE
FIX wrong contact civility

### LINKED ISSUE #11591: Bug:
# Bug

## Environment
- **Version**: 10.0.0
- **OS**: Linux
- **Web server**: Apache
- **PHP**: fpm-fcgi 5.6.40
- **Database**: MySQL or MariaDB 5.6.42-log
- **URL**: /forscoerp/htdocs/index.php?mainmenu=home&leftmenu=home

## Report
Bonjour,

j'ai installé la version 10 et j'ai perdu les civilité de mes clients et prospects, lorsque je modifie une ficher client et que je spécifie Monsieur ou Madame après l’enregistrement il n'apparait pas, donc je modifie de nouveau et le problème est toujours le même

Merci de votre aide

par avance Merci
René

### PR DESCRIPTION
**Pull request by hregis**
@eldy FIX #11591

### CODE REVIEW

File: htdocs/contact/class/contact.class.php [hunk 1]
```diff
@@ -1218,9 +1218,9 @@ public function getCivilityLabel()
 		global $langs;
 		$langs->load("dict");
 
-		$code=(! empty($this->civility_id)?$this->civility:(! empty($this->civilite)?$this->civilite:''));
-		if (empty($code)) return '';
-        return $langs->getLabelFromKey($this->db, "Civility".$code, "c_civility", "code", "label", $code);
+		if (! empty($this->civility)) return $this->civility; // civility is already defined in fetch
```
Comment (eldy) [inline htdocs/contact/class/contact.class.php]: The translation using the code must be used in priority.
The label into table is a fallback when no translation is found.
You fix seems to give the priority to the label into table

Comment (hregis) [reply htdocs/contact/class/contact.class.php]: @eldy i don't understand, civility is defined in fetch function and use the code!

`$this->civility	        = $obj->civility_code?($langs->trans("Civility".$obj->civility_code) != ("Civility".$obj->civility_code) ? $langs->trans("Civility".$obj->civility_code) : $obj->civility_code):'';`

Comment (hregis) [reply htdocs/contact/class/contact.class.php]: @without this fix the civility is not visible in contact card (view mode)

Comment (hregis) [reply htdocs/contact/class/contact.class.php]: @eldy 
look, civility is not visible

Capture d’écran 2019-08-13 à 07 38 21

Capture d’écran 2019-08-13 à 07 38 35

**Pull request closed without merging**